src/python/pants/backend/kotlin/lint/ktlint/subsystem.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants.jvm.resolve.jvm_tool import JvmToolBase
+from pants.option.option_types import SkipOption
+from pants.util.docutil import git_url
+
+
+class KtlintSubsystem(JvmToolBase):
+    options_scope = "ktlint"
+    name = "Ktlint"
+    help = "Ktlint, the anti-bikeshedding Kotlin linter with built-in formatter (https://ktlint.github.io/)"
+
+    default_version = "0.45.2"
+    default_artifacts = ("com.pinterest:ktlint:{version}",)
+    default_lockfile_resource = (
+        "pants.backend.kotlin.lint.ktlint",
+        "ktlint.default.lockfile.txt",
```

Comment:
we are using simply `.lock` for python. maybe start doing that for jvm?

Reply:
Maybe, but let's make that change all at once for JVM code (and not in this PR)?

Reply:
That's fine with me. I don't love continuing a bad pattern, but this is not blocking

Reply:
Easy enough for me to switch over to in this PR.